### DISCUSSION TITLE
What happened to the mouse and keyboard feature?

### DISCUSSION THREAD

**Discussion by lubofer** [redphx/better-xcloud#173] [General]
Hey there!
Recently, we noticed that the "1.19" update had introduced the option to use the mouse and keyboard in Xbox Cloud Gaming. However, with the "2.0.1" update, this function disappeared. What happened? Will it come back?

**Comment by redphx** [comment]
xCloud disabled this feature. I *think* I can bring it back, but since the devs don't want us to use it then I won't do it.

**Comment by lubofer** [comment]
I understand. It's a shame that they interfered with it. Would it be possible to manually install version '1.19'?"

**Reply by redphx** [reply]
It's version 2.0, not 1.19. It's still the same version in the Releases page. But since xCloud made changes on server side, that feature stopped working.
